Blocks/Persistence/file.h

```diff
@@ -302,21 +317,16 @@ class FilePersister {
       return result;
     }
 
-    Iterator& operator++() {
-      if (!valid_) {
-        CURRENT_THROW(
-            PersistenceFileNoLongerAvailable(file_persister_impl_.ObjectAccessorDespitePossiblyDestructing().filename));
-      }
+    void operator++() {
```

Comment:
Hmm, why not? :-) I'm uncertain whether it should return `bool` or `*this` to begin with.